src/plugins/arkeo/LpCard.tsx

```diff
@@ -71,7 +71,7 @@ export const LpCard: React.FC<LpCardProps> = props => {
           width='full'
           colorScheme='blue'
           mt='auto'
-          onClick={() => onClick(opportunity.assetId as OpportunityId)}
+          onClick={() => onClick((opportunity.id ?? opportunity.assetId) as OpportunityId)}
```

Comment:
Not sure why this was ever undefined, but the right side should never be evaluated - if it is, we have bigger problems.
I would remove the opt chain as this can cause bloat and hide issues, if indeed `id` was ever to get undefined/nullish (empty string)

Reply:
Note https://github.com/shapeshift/web/pull/3839 would have to go before this PR, otherwise until the migration is ran, this is valid but will become bloat once the migration goes in